language/resource-api/README.md

```diff
@@ -524,9 +507,11 @@ module Puppet::Util::NetworkDevice::Nexus
 end
 ```
 
-Inheriting from `Puppet::ResourceApi::Transport::Wrapper` will ensure that the necessary `Device` methods will be implemented using your transport.
+Inheriting from `Puppet::ResourceApi::Transport::Wrapper` ensures that the necessary `Device` methods are implemented using your transport. Specify the transport name in the `super` call to make the connection.
+
+> Note that because of the way the Resource API is bundled with Puppet agent packages, agent versions 6.0 through 6.3 are incompatible with this way of executing remote content. These versions will not be supported after [PE 2019.0 in August 2019](https://puppet.com/misc/puppet-enterprise-lifecycle).
```

Comment:
2019.0 has been out for a while? This sounds like it comes out in August 2019.

Reply:
changed to `These versions will not be supported after [support for PE 2019.0 ends in August 2019](https://puppet.com/misc/puppet-enterprise-lifecycle).` for clarity